### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by DonovanHiebert**

### CODE REVIEW

**Review by ineshjacob — APPROVED** [review]
Looks good!

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (DonovanHiebert) [inline index.html]: Suggest changes

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (DonovanHiebert) [inline index.html:48]:
```suggestion edit
</table>
```

File: index.html (hunk 1 above)
Comment (DonovanHiebert) [inline index.html]:
```suggestion
<h2 hidden>Game over, (you lose) refresh to play again 🧑‍💻 🤖!</h2>
```

**Pull request merged**